noir_stdlib/src/hash/sha256.nr

```diff
@@ -114,49 +123,40 @@ pub fn sha256_var<let N: u32>(msg: [u8; N], message_size: u64) -> HASH {
     // Pad the rest such that we have a [u32; 2] block at the end representing the length
     // of the message, and a block of 1 0 ... 0 following the message (i.e. [1 << 7, 0, ..., 0]).
     // Here we rely on the fact that everything beyond the available input is set to 0.
-    msg_block[msg_byte_ptr] = 1 << 7;
-    let last_block = msg_block;
+    // msg_block = set_block_byte(msg_block, msg_byte_ptr, 1 << 7);
```

Comment:
```suggestion
```
leftover?

Reply:
Just a reminder of how it was originally, why those methods existed, as an alternative.